specification/subscription/resource-manager/Microsoft.Subscription/preview/2020-01-01-preview/subscriptions.json

```diff
@@ -0,0 +1,469 @@
+          "description": "Error message indicating why the operation failed.",
+          "type": "string"
+        }
+      }
+    },
+    "SubscriptionName": {
+      "type": "object",
+      "description": "The new name of the subscription.",
+      "properties": {
+        "subscriptionName": {
+          "type": "string",
+          "description": "New subscription name"
+        }
+      }
+    },
+    "planTypeEnum": {
+      "type": "string",
+      "description": "the planType.",
+      "enum": [
+        "basic_support",
+        "developer_support",
+        "standard_support",
+        "prodirect_support"
+      ]
+    },
+    "operationName": {
+      "type": "object",
+      "description": "The operation Name parameter.",
+      "properties": {
+        "value": {
+          "description": "The operation Name",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DefaultSupportPlanResponseResult": {
+      "description": "Default Support plan result.",
+      "$ref": "#/definitions/SupportPlan"
+    },
+    "SupportPlanResponseResult": {
+      "description": "Support plan result.",
+      "properties": {
+        "value": {
+          "description": "The list of Support plan.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/SupportPlan"
+          }
+        },
+        "nextLink": {
+          "description": "The link (url) to the next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "SupportPlan": {
+      "description": "A SupportPlan resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SupportPlanProperties",
+          "title": "SupportPlan properties"
+        }
+      }
+    },
+    "SupportPlanProperties": {
+      "description": "The properties of the support plan.",
+      "properties": {
+        "planType": {
+          "description": "SupportPlan Type",
+          "type": "object",
+          "$ref": "#/definitions/planTypeEnum"
+        },
+        "endDate": {
+          "type": "string",
+          "description": "the end date"
+        }
+      }
+    },
+    "ProxyResource": {
+      "description": "The Resource model definition.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Id."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "PurchaseOperationStatusResponseResult": {
```

Comment:
>PurchaseOperationStatusResponseResult [](start = 5, length = 37)

not used.. i am missing something.. could you remove the ones that are not used.. ?

Reply:
clean up done